docs/ref/settings.txt

```diff
@@ -2222,16 +2230,14 @@ available in ``request.META``.)
 
 .. warning::
 
-    **You will probably open security holes in your site if you set this
-    without knowing what you're doing. And if you fail to set it when you
-    should. Seriously.**
+    **These settings can compromise site security**
```

Comment:
I think you've changed the meaning here. Both setting and not setting these can cause a security problem.